src/pages/recipe/recipe-page.tsx

```diff
@@ -27,7 +31,27 @@ export const RecipePageLoader: LoaderFunction = async ({ params }) => {
 export const HydrateRecipePage: React.FC = () => null;
 
 export const RecipePage = () => {
-    const recipe = useLoaderData<Recipe>();
+    const { recipeId } = useParams<{ recipeId: string }>();
+    const currentUserId = useAppSelector(selectUserId);
+
+    const { data: recipe } = useGetRecipeByIdQuery(recipeId!, {
```

Comment:
почитай про skipToken
```suggestion
    const { data: recipe } = useGetRecipeByIdQuery(recipeId ?? skipToken , {
```